app/controllers/manifests_controller.rb

```diff
@@ -26,4 +30,14 @@ def index
   def show
     @manifest = Manifest.find_by_uuid_or_tracking_number!(params[:id])
   end
+
+  private
+
+  def validate_manifest(content)
```

Comment:
looks like we have this same method in two places -- perhaps put in the shared concern?